Documentation/package_DesktopGL_OSX.md

```diff
@@ -0,0 +1,59 @@
+A DesktopGL game targeting OS X requires that Mono to be bundled with it. This section describes how to do it and how to create an app bundle.
+
+## Runtime requirements
+
+A DesktopGL game targeting OS X will require the following in order to run:
+# OS X Snow Leopard 10.6 or above
+# OpenGL 4.0 compliant graphics card and driver
+
+## Preparing your game files
+
+### Getting and configuring MonoKickstart
+
+### Additional dependencies
+
+If your game uses other libraries than MonoGame, verify if they make calls to native libraries. A good way to know it, is to try running the .app bundle on an OS X without the Mono SDK installed. If there is any native library missing, the game will run into a DllNotFoundException like this one:
+
+Unhandled Exception: System.TypeInitializationException: An exception was thrown by the type initializer for  System.Drawing.GDIPlus ---> System.DllNotFoundException: gdiplus.dll
+
+To add this native dependency to your MonoKickstart configuration, you will have to first edit the monoconfig file by adding the mapping to the OS X equivalent:
+
+	<dllmap dll="gdiplus.dll" target="libgdiplus.dylib" os="osx"/>
```

Comment:
I did (in the case of GDI+), I think it still applies if developers use libraries other than pure MonoGame in their projects.

I should also document which .Net libraries not to use because they would require special mapping and may not be supported by Mono at all (e.g. kernel32, user32, System.IO.Compression...).